OlinkAnalyze/R/olink_qc_plot.R

```diff
@@ -52,6 +53,9 @@ olink_qc_plot <- function(df, color_g = "QC_Warning", plot_index = F, label_outl
 
   npx_df_qr <- npx_df %>%
     dplyr::group_by(Panel, SampleID, Index) %>%
+    dplyr::mutate(QC_Warning = dplyr::if_else(all(toupper(QC_Warning) == 'PASS'),
+                                              'PASS',
+                                              'WARNING')) %>%
```

Comment:
Also here, suggestion to keep "Title case"
```suggestion
    dplyr::mutate(QC_Warning = dplyr::if_else(all(toupper(QC_Warning) == 'PASS'),
                                              'Pass',
                                              'Warning')) %>%
```